### PR TITLE
MNT: add user-customizable identifier "ledName" to FB_LED

### PR DESCRIPTION
**Pull request by Mbosum**
## Description

Added a String Pragma as a VAR_INPUT to the FB_LED.

## Motivation and Context

I would like for there to be a description PV which will show up on the typhos screen, which will be user editable so users can label their LEDs.

## How Has This Been Tested?

I have not been able to test this anywhere.

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive comments
- [ ] Test suite passes locally
- [x] Libraries are set to ``Always Newest`` version (``Library, *``)
- [ ] Committed with ``pre-commit`` or ran ``pre-commit run --all-files``

### CODE REVIEW

**Comment by Mbosum** [comment]
@ZLLentz @klauer before I do all the pre-merge stuff, I just wanted to get your opinion on the String pragma.

File: LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED.TcPOU [hunk 1]
```diff
@@ -21,6 +21,12 @@
     2022-5-03 Maarten Thomas-Bosum
 *)
 VAR_INPUT
+	{attribute 'pytmc' := '
+        pv: NAME
+        io: io
```
Comment (klauer) [inline LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED.TcPOU:26]: I'd recommend adding something like this to the pragma:
```
field: DESC Descriptive name for the LED
autosave_pass0: VAL DESC
```

Autosave reference: https://pcdshub.github.io/pytmc/v2.9.1/pragma_usage.html#autosave

Comment (Mbosum) [reply LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED.TcPOU:26]:
```
{attribute 'pytmc' := '
        pv: NAME
        io: io 
	field: DESC Descriptive name for the LED
        autosave_pass0: VAL DESC
    '}
	ledName : STRING;
```

@klauer So this is what you had in mind?

Comment (klauer) [reply LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED.TcPOU:26]: (I had this tab open and forgot to reply, sorry @Mbosum )

Yes, that's what I was suggesting 👍

Comment (Mbosum) [reply LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED.TcPOU:26]: Great, thanks for answering!

**Review by klauer — APPROVED** [review]

**Review by ZLLentz — APPROVED** [review]
LGTM! Probably should have merged since reviewer here was on vacation

**Comment by klauer** [comment]
It goes lost in my queue, sorry @Mbosum!

**Pull request merged**